src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -33,7 +33,11 @@ public class GpioController : IDisposable
     /// If a pin element exists, that pin is open. Uses current controller's numbering scheme
     /// </summary>
     private readonly ConcurrentDictionary<int, PinValue?> _openPins;
-    private readonly ConcurrentDictionary<int, GpioPin> _gpioPins;
+
+    /// <summary>
+    /// The GPIO pins open in the GpioController.
+    /// </summary>
+    protected readonly ConcurrentDictionary<int, GpioPin> _gpioPins;
```

Comment:
If we want to include it in the next release, let's mark this as experimental. Also, if this will be renamed to be protected, we should rename to not have an `_` as suffix, since that is convention for private fields.